common/src/main/java/com/discordsrv/common/config/main/OnlineSyncConfig.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * This file is part of DiscordSRV, licensed under the GPLv3 License
+ * Copyright (c) 2016-2025 Austin "Scarsz" Shapiro, Henri "Vankka" Schubin and DiscordSRV contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package com.discordsrv.common.config.main;
+
+import com.discordsrv.common.abstraction.sync.enums.SyncDirection;
+import com.discordsrv.common.config.main.generic.AbstractSyncConfig;
+import com.discordsrv.common.util.Game;
+import org.spongepowered.configurate.objectmapping.meta.Comment;
+
+public class OnlineSyncConfig extends AbstractSyncConfig<OnlineSyncConfig, Game, Long> {
```

Comment:
We don't want the direction to be changeable for this, but the setting will be inherited from AbstractSyncConfig

I don't really have any good ideas on how to solve this besides just copying the options to a new config model and just have intermediary like group sync does https://github.com/DiscordSRV/Ascension/blob/a5b5a5ae64468aef251bec5188bc710085bcc0d4/common/src/main/java/com/discordsrv/common/config/main/GroupSyncConfig.java#L80